src/features/auth/components/Donation.tsx

```diff
@@ -0,0 +1,213 @@
+  onDonateAndPlayClick: (address: CharityAddress) => void;
+}
+
+const CharityCarouselItem = ({
+  url,
+  name,
+  info,
+  address,
+  onDonateAndPlayClick,
+}: CharityCarouselItemProps) => {
+  const onAboutClick = (url: string) => {
+    window.open(url);
+  };
+  return (
+    <OuterPanel className="flex-col inline-flex items-center justify-center w-full">
+      <div className="flex flex-col items-center mb-3 whitespace-normal">
+        <h5 className="text-sm text-shadow underline mb-3">{name}</h5>
+        <p className="text-xs text-center text-shadow mb-2">{info}</p>
+      </div>
+
+      <div className="flex w-full">
+        <Button className="w-full mr-1" onClick={() => onAboutClick(url)}>
+          <span className="text-xs mr-1">About</span>
+          <img src={question} className="scale-110" alt="question-mark" />
+        </Button>
+        <Button
+          className="w-full ml-1"
+          onClick={() => onDonateAndPlayClick(address)}
+        >
+          <span className="text-xs whitespace-nowrap">Donate & Play</span>
+        </Button>
+      </div>
+    </OuterPanel>
+  );
+};
+
+interface Props {
+  onDonate: (charityAddress: CharityAddress, donation: number) => void;
+}
+
+export const Donation: React.FC<Props> = ({ onDonate }) => {
+  const [donation, setDonation] = React.useState(1.0);
+
+  const onDonationChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setDonation(roundToOneDecimal(e.target.valueAsNumber));
+  };
+
+  const incrementDonation = () => {
+    setDonation((prevState) => roundToOneDecimal(prevState + 0.1));
+  };
+
+  const decrementDonation = () => {
+    if (donation === 0.1) {
+      setDonation(0.1);
+    } else setDonation((prevState) => roundToOneDecimal(prevState - 0.1));
+  };
+
+  const onDonateAndPlayClick = (charityAddress: CharityAddress) => {
+    onDonate(charityAddress, donation);
+  };
+
+  return (
+    <>
+      <div className="flex flex-col text-shadow items-center">
+        <h2 className="text-base mb-1">Donate to play.</h2>
+        <p className="text-xs mb-3 text-center">
+          To start a farm, we require a minimum donation of 1 Matic to support
+          the operating costs of Sunflower Land.
+        </p>
+        <p className="text-xs mb-3 text-center">
+          10% of this donation will go to a charity of your choice.
+        </p>
+      </div>
+      <div className="flex flex-col items-center mb-3">
+        <div className="relative">
+          <input
+            type="number"
+            className="text-shadow shadow-inner shadow-black bg-brown-200 w-24 p-1 text-center"
+            step="0.1"
+            min={1.0}
+            value={donation}
+            onChange={onDonationChange}
+          />
+          <img
+            src={upArrow}
+            alt="increment donation value"
+            className="absolute -right-4 top-0"
+            onClick={incrementDonation}
+          />
+          <img
```

Comment:
Consider hiding this when the value = 1 . Currently users can give a lower donation

<img width="229" alt="Screen Shot 2022-02-01 at 10 26 40 am" src="https://user-images.githubusercontent.com/11745561/151976881-e546e945-b360-4935-aaa3-c99e5ed3e5a5.png">